macroquad_macro/src/lib.rs

```diff
@@ -34,12 +34,19 @@ pub fn main(attr: TokenStream, item: TokenStream) -> TokenStream {
     }
     modified.extend(source);
 
+    let (method, ident) = match attr.into_iter().next() {
```

Comment:
Could you make separate PR with just macro changes? 
It would be way easier to review!

Reply:
Ok

Reply:
Move macro changes to https://github.com/not-fl3/macroquad/pull/28 PR